### DISCUSSION TITLE
Error to setup jenkins artillery engines playwright

### DISCUSSION THREAD

**Discussion by jodybarata** [artilleryio/artillery#1696] [Artillery]
Hello, Canyou give me an example jenkinsfile for artillery engines playwright? because i implement on local always return message ` Cannot run program "nohup"`

**Comment by hassy** [comment]
hi @jodybarata 👋 

That looks like a Jenkins/environment-specific issue from this SO thread: https://stackoverflow.com/questions/45140614/jenkins-pipeline-sh-fail-with-cannot-run-program-nohup-on-windows

Are you able to run plain Playwright tests in Jenkins?

This video might be helpful:

https://www.youtube.com/watch?v=pm1FJq-2bjo
